BackofficeBundle/Form/Type/SiteAliasType.php

```diff
@@ -22,8 +22,8 @@ public function __construct($siteAliasClass)
     {
         $this->siteAliasClass = $siteAliasClass;
         $this->schemeChoices = array(
-            SchemeableInterface::SCHEME_HTTP => SchemeableInterface::SCHEME_HTTP,
-            SchemeableInterface::SCHEME_HTTPS => SchemeableInterface::SCHEME_HTTPS
+            SchemeableInterface::SCHEME_HTTP => 'open_orchestra_backoffice.http',
```

Comment:
Maybe you should inject a sublevel in the translation key : open_orchestra_backoffice.scheme.http and the same for https